students/srmorehouse/Lesson02/print_grid.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+import argparse
+
+#
+# printHorizontal
+#
+# start the box printing on the first line
+#
+def printHorizontal(totalNumberOfBoxes,sizeOfEachBox):
```

Comment:
make sure to use snake_case for function and variable names -> `print_horizontal`